main.rb

```diff
@@ -0,0 +1,152 @@
+# test.rb
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './book'
+# rubocop:disable all
+# Create class App
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+
+    person_type = gets.chomp
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      return
+    end
+
+    print 'Age: '
+
+    age = gets.chomp
+
+    print 'Name: '
+
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        # *If needed, ask for parameters for the option.
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
```

Comment:
- [ ] This method is not saving the student name properly, whenever I create a student i will get false instead of the name, please fix this issue. 🐱 

    ![imagen](https://user-images.githubusercontent.com/60789317/141517473-c672ac39-2d97-408e-a8e0-1a26cae042ed.png)

Reply:
This issue has been addressed well. Great job! :clap: :clap: